lib/elasticity/index_config.rb

```diff
@@ -26,7 +29,9 @@ def client
 
     def definition
       return @definition if defined?(@definition)
-      @definition = { settings: @elasticity_config.settings, mappings: { @document_type => @mapping || {} } }
+      @definition = {
+        settings: settings || {}, mappings: { @document_type => @mapping || {} }
```

Comment:
would we want to merge `@elasticity_config.settings` and `@settings`, making `@elasticity_config.settings` the defaults, and allowing keys in `settings` to override?